code_folder/A1_csv_to_json.py

```diff
@@ -0,0 +1,356 @@
+import pandas as pd
+import os
+import json
+from pathlib import Path
+import logging
+
+"""
+How to use:
+all default input csv"s are stored in "/mvs_eland/inputs/elements/default_csv".
+These csvs are not to be changed! All csvs that you need in order to set up 
```

Comment:
Have you tested what happens if an asset type is not existing, eg. if I have a project that does not have storage or diesel generators? This scenario should run through - either accepting that that specific csv is missing completely or accepting that the existing csv does not have any colums defining an asset.

Reply:
The code does not complain if a csv file is missing. I thought about making a list of required csv files, because I guess the "project.csv" and others are essential for mvs to function. What do you think?
If a whole collumn is missing, the respective json dictionary is not created, because it parses through all collumns. So again, this component won't show in the final json. Up to now no error message is implemented for that case.

Reply:
> The code does not complain if a csv file is missing. I thought about making a list of required csv files, because I guess the "project.csv" and others are essential for mvs to function. What do you think?
Sounds perfect! Please also include a logging message: Depending how you see it a logging.debug('No csv file xy found. This is an accepted option.') 
or logging.warning('No csv file xy found. Are you sure?')
or logging.error('Required input file xy is missing! Please add it to the inputs.") 
> If a whole collumn is missing, the respective json dictionary is not created, because it parses through all collumns. So again, this component won't show in the final json. Up to now no error message is implemented for that case.
A message logging.debug('No energyProduction assets are added as energyProduction.csv asset columns are empty.')

Reply:
ok. what are the required files? Is this correct?:
required_files_list=['energyProduction', 'project', 'fixcost',
                              'simulation_settings', 'project_data',
                              'economic_data']

Reply:
I added a general logging.debug message if the columns of a file are empty (except of "unit").

Reply:
Solved this in #68